packages/itwin/tree-widget/src/test/trees/category-tree/CategoriesTree.test.tsx

```diff
@@ -289,6 +296,106 @@ describe("CategoryTree", () => {
         });
       });
 
+      describe("Hierarchy level filtering", () => {
```

Comment:
```suggestion
      describe("hierarchy level filtering", () => {
```

Same in other tree tests